flask/db/seed_data.py

```
@@ -0,0 +1,255 @@
+    def _validate_data(self, cursor):
+        """Validate that the seed data was inserted correctly."""
+        validations = [
+            ("users", "SELECT COUNT(*) FROM users", 5, "users"),
+            ("movies", "SELECT COUNT(*) FROM movies", 6, "movies"),
+            ("showtimes", "SELECT COUNT(*) FROM showtimes", 14, "showtimes"),
+            ("reservations", "SELECT COUNT(*) FROM reservations", 4, "reservations"),
+        ]
+        
+        validation_passed = True
+        
+        for name, query, expected, description in validations:
```

Ruff (0.11.9)

Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)